docs/tutorial.rst

```diff
@@ -349,7 +349,13 @@ the view which will do ``Country.objects.create(name='Tibet')``. It will be
 included in the server response so that the script can add it to the widget.
 
 Note that creating objects is only allowed to staff users with add permission
```

Comment:
Should we change this so that we don't need to be staff user anymore ?

Reply:
The point is. Staff members get this privilege by default; to grant same right to other users (not staff members) it's necessary to set the `add` permission explicitly.